src/context/fetchCapitalCities.jsx

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+async function fetchCapitalCities() {
+  const response = await axios.get("https://restcountries.com/v3.1/all?fields=capital");
+
+  return response;
+}
+
+export default fetchCapitalCities;
```

Comment:
Why are we keeping these files in the `/context` folder? `context` suggests that it should have `React.Context` (providers etc.) in it.

I suggest we change `context` to `api`. What do you think?

Reply:
Make sense. I was thinking to use `useContext` later. But yeah at this stage, we can name it `api` for now.